packages/ui-library/src/components/input-field-text/index.ts

```diff
@@ -170,6 +170,7 @@ export class BlrInputFieldText extends LitElementCustom {
       });
 
       const inputClasses = classMap({
+        'error-input': this.hasError || false,
```

Comment:
this is also missing the `disabled` class. currently the placeholder has the wrong color in disabled state when compared to design.

Reply:
resolved in the latest commit. Thank you